### PR TITLE
update changelog for fissix

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by takluyver** [comment]
You can see from the prior state of it what a poor job I've been doing of managing the changelog, but if you want to update it, there are a few other PRs which got merged since 0.7: https://github.com/python-modernize/python-modernize/commits/master

**Comment by graingert** [comment]
https://github.com/python-modernize/python-modernize/compare/0.7...0.8rc1

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,12 @@
-Unreleased
-==========
+Version 0.8rc1
+==============
+
+Released 2020-07-20.
+
+* use fissix istead of deprecated lib2to3
```
Comment (graingert) [inline CHANGELOG.rst:6]:
```suggestion
* use fissix istead of deprecated lib2to3
* and other minor fixes https://github.com/python-modernize/python-modernize/compare/0.7...0.8rc1
```

Comment (graingert) [reply CHANGELOG.rst:6]: @takluyver I don't quite have the bandwidth to decode all the commits/prs into a nice changelog, but I think the fissix change is important to get into the changelog before 0.8 final

will this cop-out do for now?

Comment (takluyver) [reply CHANGELOG.rst:6]: Given that I've been neglecting the changelog entirely, I can't really complain. :wink:

**Pull request merged**